Source/WebCore/PAL/pal/spi/cocoa/WritingToolsSPI.h

```diff
@@ -29,6 +29,7 @@
 
 #if USE(APPLE_INTERNAL_SDK)
 
+#import <UIKit/UIKit.h>
```

Comment:
This needs to be guarded by `#if PLATFORM(IOS_FAMILY)`. EWS won't catch this since `ENABLE(WRITING_TOOLS)` isn't true on iOS 17.